src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/EvaluationContext.cs

```diff
@@ -158,6 +160,25 @@ internal sealed class EvaluationContext : EvaluationContextBase
             Guid moduleVersionId,
```

Comment:
We could merge the two overloads. The one difference is that this second overload is used for compilations that contain referenced modules only (see caller). It is unfortunate to require each overload to call NormalizeILOffset but the compiler will report cast errors between uint and int if that's not done. Will leave as two overloads for now.